PyPDF2/errors.py

```diff
@@ -33,4 +33,16 @@ class ParseError(Exception):
     pass
 
 
+class WrongPasswordError(PdfReadError):
```

Comment:
From the names only, I would expect the wrong password exception to be subclass ING the the FileNotDecryptedError

Reply:
> From the names only, I would expect the wrong password exception to be subclass ING the the FileNotDecryptedError

updated to address this. thanks!